checkov/arm/graph_builder/local_graph.py

```diff
@@ -121,28 +123,30 @@ def _create_resource_vertices(self, file_path: str, resources: list[dict[str, An
                     path=file_path,
                     block_type=BlockType.RESOURCE,
                     attributes=attributes,
-                    id=f"{resource_type}.{resource_name}",
+                    id=f"{resource_type}.{resource_name}"
                 )
             )
 
     def _create_edges(self) -> None:
-        self._create_vars_and_parameters_edges()
-        # todo add explicit references edges
-
-    def _create_edge(self, element_name: str, origin_vertex_index: int, label: str) -> None:
-        vertex_name = element_name
-        if "." in vertex_name:
-            # special case for bicep and arm elements, when properties are accessed
-            vertex_name = vertex_name.split(".")[0]
-
-        dest_vertex_index = self.vertices_by_name.get(vertex_name)
-        if dest_vertex_index or dest_vertex_index == 0:
-            if origin_vertex_index == dest_vertex_index:
-                return
-            edge = Edge(origin_vertex_index, dest_vertex_index, label)
-            self.edges.append(edge)
-            self.out_edges[origin_vertex_index].append(edge)
-            self.in_edges[dest_vertex_index].append(edge)
+        for origin_vertex_index, vertex in enumerate(self.vertices):
+            if 'dependsOn' in vertex.attributes:
```

Comment:
please use a const value parameter for it (like `DEPENDS_ON_FIELD`)

Reply:
Done